src/controllers/quote.js

```diff
@@ -109,6 +112,25 @@ const model = require('../models/quote')
 /* eslint-enable */
 
 exports.get = function * () {
+  const queryParams = this.query
```

Comment:
Maybe have a `function validateAmounts (query.source_amount, query.destination_amount)`, just to make the `get` function a bit smaller